recipes/etc2comp/all/test_package/test_package.cpp

```diff
@@ -0,0 +1,16 @@
+#include <iostream>
+
```

Comment:
```suggestion
#include <cstdlib>

```

EXIT_SUCCESS is part of cstdlib. There is no invocation for iostream here.